frontend/src/fileReceiver.js

```diff
@@ -62,7 +62,7 @@ class FileReceiver extends EventEmitter {
         });
       })
       .then(([fdata, key]) => {
-        this.emit('decrypting', true);
+        this.emit('decrypting');
```

Comment:
Although I don't think it affects the outcome of the tests, I think we should change the frontend tests to reflect that these listeners only get called once.